lib/create-http-client.js

```diff
@@ -37,9 +43,19 @@ export default function createHttpClient (axios, options) {
   const config = Object.assign({}, defaultConfig, options)
 
   if (!config.accessToken) {
-    throw new Error(`'accessToken' option is missing`)
+    throw new TypeError(`'accessToken' option is missing`)
+  }
+
+  // Set up log & error listeners
+  if (config.logHandler) {
+    config.logEmitter.addListener('log', config.logHandler)
```

Comment:
is there a reason we user can pass in the event and the handler, wouldn't be nicer if the user just can pass his event emitter instance and handle it at app level?